internal/pkg/config/config.go

```diff
@@ -268,33 +268,38 @@ func (pm packageMatcher) match(pkg *types.Package) bool {
 	return pm.PackageNameRE.MatchString(pkg.Path())
 }
 
-type NameMatcher struct {
-	PackageRE regexp.Regexp
-	TypeRE    regexp.Regexp
-	MethodRE  regexp.Regexp
+type methodMatcher struct {
```

Comment:
Maybe just `matcher`, since this can match more than just methods?

Reply:
Actually, maybe `callMatcher` would be more expressive, since that is what we are trying to match in the end?

Reply:
Renamed to `callMatcher`.